src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstLag;
```

Comment:
The variable name `firstLag` might be confusing. Consider renaming it to `base` or `height` for better clarity and understanding, especially since this is a right triangle.